pydash/pydash_web/blueprint.py

```diff
@@ -0,0 +1,12 @@
+"""
+Serves as a blueprint for the entire pydash_web package.
+url_for() calls within this package should prepend 'pydash_web.' to their input argument.
+  [e.g. url_for(login) becomes url_for(pydash_web.login) ]
+route decorators in this package should also use this blueprint object instead of the flask application object.
+"""
+
+from flask import Blueprint
+
+bp = Blueprint('pydash_web', __name__)
+
+from pydash_web import routes
```

Comment:
unused import?

Reply:
routes have to be imported here in order to link the routes to the blueprint, if I recall correctly.